docs/spec/src/integration/spec/6-secure-integration.md

```diff
@@ -1,29 +1,44 @@
+2. Cert Validation - ensures sufficient operator stake has signed to make the blob available, for all specified quorums. The stake is obtained onchain at a given reference block number (RBN) specified inside the cert.
+3. Blob Validation - ensures that the blob used is consistent with the KZG commitment inside the Cert.
 
-The validation process is:
+### 1. RBN Recency Validation
 
-1. Validate the Cert (against state on ethereum)
-2. Validate the Blob (against the Cert)
+First, we assume that every rollup stack has a mechanism to force blobs to be included in the rollup’s batcher-inbox in a timely matter, such that the cert's blob is still available on EigenDA for long enough to cover the fraud proof duration (typically 7 days), or long enough to be downloadable by zk rollup participants.
 
-### Cert Validation
+> For example, optimism has a [sequencerWindow](https://docs.optimism.io/stack/rollup/derivation-pipeline#sequencer-window) which forces batches to land onchain in a timely fashion (12h). This indirectly also forces EigenDA certs to land onchain in a timely fashion. Therefore, it is safe to assume that the certs land onchain < SequencerWindowSize after having been signed.
+>
+> OP stack rollups thus need to make sure that `SequencerWindow` << 14 days.
 
-Cert validation is done inside the EigenDACertVerifier contract, which EigenDA deploys as-is, but is also available for rollups to modify and deploy on their own. Specifically, [verifyDACertV2](https://github.com/Layr-Labs/eigenda/blob/98e21397e3471d170f3131549cdbc7113c0cdfaf/contracts/src/core/EigenDACertVerifier.sol#L86) is the entry point for validation. This could either be called during a normal eth transaction (either for pessimistic “bridging” like EigenDA V1 used to do, or when uploading a Blob Field Element to a one-step-proof’s [preimage contract](https://specs.optimism.io/fault-proof/index.html#pre-image-oracle)), or be zk proven using a library like [Steel](https://github.com/risc0/risc0-ethereum/blob/main/crates/steel/docs/what-is-steel.md).
+This guarantee alone however is not enough, because the EigenDA disperser could have chosen a reference block number (RBN) that is very old, such that the stake of operators does not represent the true current value. Because EigenDA V2 doesn't pessimistically bridge certs onchain like EigenDA V1, enforcing RBN to not be too old is thus the responsibility of the rollup.
 
-The [cert verification](https://github.com/Layr-Labs/eigenda/blob/master/contracts/src/libraries/EigenDABlobVerificationUtils.sol#L162) logic consists of:
+> Rollups must enforce that `cert.RBN` > `cert.L1InclusionBlock` - `RecencyWindowSize`
 
-1. [merkleInclusion](https://github.com/Layr-Labs/eigenda/blob/c3d1ff2a9a1ec39fc78c4936eabbc2443df38571/contracts/src/libraries/EigenDABlobVerificationUtils.sol#L173): 
+We suggest using the same value as the `SequencerWindowSize` for the `RecencyWindowSize`, namely 12h.
```

Comment:
Hmm, are we sure about this?

When the disperser chooses an `RBN` for a batch, it intentionally chooses one that is slightly in the past (see the [finalization delay](https://github.com/Layr-Labs/eigenda/blob/fc9a97144585186f25bcdc77134d02a6de9d53f0/disperser/controller/dispatcher.go#L32))

If `RecencyWindowSize == SequencerWindowSize`, in a practical sense you're decreasing the `SequencerWindowSize` by the disperser's `finalization delay`, since a batch created at `now - SequencerWindowSize + 1` will be invalid, even though it's a perfectly normal batch, and was submitted within the required sequencer window.

Would it make more sense for `RecencyWindowSize` to be some moderate amount > `SequencerWindowSize`?

Reply:
But so... seems like we need to make `RecencyWindow = SequencerWindowSize + how_far_back_disperser_can_set_RBN`... and even if we do that there's nothing stopping disperser from changing its behavior and setting RBN to be `RecencyWindowSize` in the past...

Actually I think we can solve this by changing the validity equation to be:
```
cert.ReferenceBlockNumber < cert.L1InclusionBlock <= max(SequencerWindowSize, cert.ReferenceBlockNumber + eigenDAInclusionWindowSize)
```
WDYT?

Reply:
Couple questions:

- When you say `eigenDAInclusionWindowSize`, do you actually mean `recencyWindowSize`?
- The first param of your `max` function is `SequencerWindowSize`, but I assume you meant to add something to it?

Reply:
Yes and yes. Sorry copy-pasted from somewhere else and was careless. updated with correct variable names.

Reply:
Ok so think the overall TLDR of this discussion we had last week was "yes it makes sequencerWindowSize a bit shorter, but who cares". If you're happy with this feel free to resolve this discussion.